src/main/java/org/jboss/pnc/reqour/dto/ExternalToInternalDTO.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2024-2024 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.reqour.dto;
+
+import jakarta.validation.constraints.Null;
+import lombok.Builder;
+import lombok.Value;
+import org.jboss.pnc.reqour.common.validation.ValidationGroups;
+
+@Value
+@Builder(toBuilder = true)
+public class ExternalToInternalDTO {
+
+    String externalUrl;
```

Comment:
**Note:** In the current implementation of repour, snake-case is used. However, I believe we should get rid of that, right?

Reply:
I've created `reqour.dto` package inside of the `pnc-api`, i.e. not rewriting the old, still used `repour.dto`.
I find it as a reasonable compromise, since it allows us to get rid of the snake-case inside of the new reqour (and also use `pnc-api` from there). On the other hand, we are not modifying used `repour.dto` package.

Reply:
I've created `reqour.dto` package inside of the `pnc-api`, i.e. not rewriting the old, still used `repour.dto`.
I find it as a reasonable compromise, since it allows us to get rid of the snake-case inside of the new reqour (and also use `pnc-api` from there). On the other hand, we are not modifying used `repour.dto` package.

See: https://github.com/project-ncl/pnc-api/pull/216